### ISSUE TITLE
[Overlay] there is no 'Duplicate the conversation to be able to edit it' button in shared and published chat when overlayConversationId is set

### ISSUE THREAD

**Issue by VolhaBazhkova** [epam/ai-dial-chat#2851]
### EPAM AI DIAL chat version

0.22.0

### What happened?

Precondition: 
1. User has chat1 in Shared with me section; chat2 in Organization section
2. Enable features:
    Feature.ConversationsPublishing,
    Feature.ConversationsSharing
    Set overlayConversationId

Steps: User clicks on Chat1/Chat2 -> there is no 'Duplicate the conversation to be able to edit it' button at the bottom of the chat -> **bug**

Note: reproduced locally in overlay Sandbox
Note: everything is ok if there is no overlayConversationId

### Confidential information

- [X] I confirm that do not share any confidential information

**Comment by denys-kolomiitsev** [comment]
This is expected behavior by design. 
When `overlayConversationId` is provided user should not be able to switch to another conversation.

**Comment by VolhaBazhkova** [comment]
Agree. According to the idea of the feature overlayConversationID, the bug should be treated like misconfiguration.
